### DISCUSSION TITLE
OIDC : Version 3.2.10.Final Swagger UI does not refresh once the JWT token expires

### DISCUSSION THREAD

**Discussion by kirankolli-addepar** [quarkusio/quarkus#39119] [Q&A]
When first loaded the Swagger UI prompts to login , however even after logging in the Swagger UI shows up still as 'Unauthorized' . This then requires users to re-login again on the Swagger page and surprisingly also shows a list of roles available to choose when logging in.
Post login and once the token expires if you hit 'Refresh' on the Swagger UI the page shows 'Coonection refused' error.
![image](https://github.com/quarkusio/quarkus/assets/122354105/1239f7d1-b17f-4019-a7dd-13cb397a0515)

Reproducer project attached.
[code-with-quarkus 6.zip](https://github.com/quarkusio/quarkus/files/14464207/code-with-quarkus.6.zip)

[a comment by an automated account is not shown]

**Comment by sberyozkin** [comment]
@kirankolli-addepar As I've explained in the other discussion, Swagger UI does not integrate with OIDC `web-app` applications. So I'm not sure what can be done here
